### DISCUSSION TITLE
User-curated emote "theme" collections

### DISCUSSION THREAD

**Discussion by jupjohn** [SevenTV/Extension#216] [Ideas]
Not entirely sure how an implementation would look, but the ability for emote creators to group their emote into a user-curated "theme" or "category" would help improve discoverability of similar emotes. 

Example:
"pogg emotes":
  "hehe" (https://7tv.app/emotes/6150c69843b2d9da0d329847)
  "pogg" (https://7tv.app/emotes/61500f246251d7e000daa3ca)
  "majj" (https://7tv.app/emotes/61359284baf761f5d4fd6ea1)

Alternative usage would be for seasonal emote sets. Users could create a Christmas, Halloween etc. emote theme of their current emote set and be able to swap them in and out from those lists.

**Comment by AnatoleAM** [comment]
This is already implemented and will release with the rest of API V3 (in the form of Emote Sets)
